test/extended/router/metrics.go

```diff
@@ -257,7 +257,7 @@ var _ = g.Describe("[sig-network][Feature:Router]", func() {
 			}()
 
 			o.Expect(wait.PollImmediate(10*time.Second, 5*time.Minute, func() (bool, error) {
-				contents, err := getBearerTokenURLViaPod(ns, execPod.Name, fmt.Sprintf("%s/api/v1/targets", prometheusURL), token)
+				contents, err := getBearerTokenURLViaPod(ns, execPod.Name, fmt.Sprintf("%s/api/v1/targets?state=active", prometheusURL), token)
```

Comment:
This belongs in its own commit with a comment describing why it was necessary.

Reply:
It's already merged https://github.com/openshift/origin/commit/0fbe9178794c72636b372d10d6132b6fac82b372